jplag/src/main/java/de/jplag/options/JPlagOptions.java

```diff
@@ -98,7 +98,7 @@
      * of the compatibility fallback listed above.
      * </p>
      */
-    private Optional<String> baseCodeSubmissionName = Optional.empty();
+    private String baseCodeSubmissionName = null;
```

Comment:
What is the benefit of switching to the type internally instead of the optional?

Reply:
Optional was designed to provide a limited mechanism for library method return types in which a clear way to represent "no result"  was needed.

Taken from here https://github.com/JetBrains/intellij-community/blob/master/plugins/InspectionGadgets/src/inspectionDescriptions/OptionalUsedAsFieldOrParameterType.html :)

Reply:
I think I did not even find all Optionals :(